setup.cfg

```
@@ -9,6 +9,7 @@ maintainer_email = hello@upcloud.com
 url = https://github.com/UpCloudLtd/upcloud-python-api
 packages=['upcloud_api', 'upcloud_api.cloud_manager']
 license = MIT
+license_files = []
```

Fixes issue with twine, https://github.com/pypa/twine/issues/1216#issuecomment-2609745412

---

Don't know if it makes much of a difference, but I suppose we could add the actual license file here too.
```suggestion
license_files = LICENSE.txt
```
This syntax is untested, but I suppose it's the correct one per https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#specifying-values. `packages` above uses something else that doesn't seem to be noted in the config spec.